website/pages/docs/deployment/kubernetes.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Deploying CloudQuery to a Kubernetes
+description: In this tutorial we will set up CloudQuery on a Kubernetes as a CronJob.
+tag: tutorial
+date: 2023/07/20
+image: https://i3.ytimg.com/vi/3dJVj6JjHqY/maxresdefault.jpg
+video: https://youtu.be/3dJVj6JjHqY
+---
+
+import { Callout } from 'nextra-theme-docs';
+
+# Deploying CloudQuery using Kubernetes CronJobs
+
+In this tutorial we will set up a Kubernetes CronJob to run CloudQuery Sync on a regular schedule.
+Due to its flexibility and standardization across all cloud providers, Kubernetes is the #1 way that DevOps & Platform Engineers prefer to deploy workloads and microservices.
```

Comment:
Maybe keep `Kubernetes is the #1...` out? and keep it a bit more opinionated.

Reply:
I feel that the search numbers support the claim, but I'm happy to change it.
Would this work better for you?
```suggestion
Due to its flexibility and standardization across cloud providers, Kubernetes is commonly used by DevOps & Platform Engineers when deploying workloads and microservices.
```